cpovc_forms/views.py

```diff
@@ -8963,7 +8963,9 @@ def case_plan_template(request, id):
 
         if my_request:
             caseplandata = json.loads(my_request)
+            print("kkkkkkkkkkkk",caseplandata)
```

Comment:
@abdy254 Please remove all print statements from this file and make another request as I approve this updates